.github/workflows/node.js.yml

```diff
@@ -17,6 +17,7 @@ jobs:
           node-version: 14.x
       - uses: microsoft/playwright-github-action@v1
       - run: npm ci
+      - run: npm run get-repc
```

Comment:
I removed the get-repc from npm install flow since we already include the wasm files in the npm package.

Reply:
Why does github need it?

Reply:
github action runs from the git checkout. Not by installing the npm package

Reply:
I should use `npm prepare`. That gets called when doing `npm install` (but not when doing `npm install foo`) as well as when doing `npm publish` and `npm pack`